### ISSUE TITLE
'av_scan_background' has no UI entry where it can be changed

### ISSUE THREAD

**Issue by mmattel** [owncloud/files_antivirus#262]
Is this intended that the value can only be changed via
``php occ config:app:set files_antivirus av_scan_background --value 'false'`` ?

Compared to eg: ``'av_mode'`` in templates/settings.php
```
<h2><?php p($l->t('Antivirus Configuration'));?></h2>
 <p class="av_mode">
 <label for="av_mode"><?php p($l->t('Mode'));?></label>
 <select id="av_mode" name="avMode">
 <?php print_unescaped(html_select_options(['executable' => $l->t('Executable'), 'daemon' => $l->t('Daemon'), 'socket' => $l->t('Daemon (Socket)')],
```
See also: https://github.com/owncloud/files_antivirus/pull/213#issuecomment-365598066

@VicDeo @patrickjahns

**Comment by mmattel** [comment]
@VicDeo mind to support?
Somehow important as this also impacts the commad line documentation we are working on.
Things should be complete.

**Comment by jnweiger** [comment]
Confirmed fixed in 1.2.0-rc.1

**Comment by jnweiger** [comment]
Confirmed fixed in 1.2.0-rc.1
